src/cheevos/Cheevos.cpp

```diff
@@ -129,6 +135,48 @@ void CCheevos::EvaluateRichPresence(std::string& evaluation, unsigned int consol
   evaluation = _evaluation;
 }
 
+void CCheevos::ActivateAchievement(unsigned cheevo_id, const char* memaddr)
+{
+  rc_runtime_activate_achievement(&m_runtime, cheevo_id, memaddr, NULL, 0);
+  // it will return integer value 0 in case achivement is activated successfully.
```

Comment:
maybe we should add a check here instead of a comment like we do in the other methods